eventloop/eventloop.go

```diff
@@ -263,6 +268,12 @@ func (loop *EventLoop) addTimeout(f func(), timeout time.Duration) *Timer {
 }
 
 func (loop *EventLoop) addInterval(f func(), timeout time.Duration) *Interval {
+	// If no timeout is provided set timeout to 1ms. Follows node documentation.
```

Comment:
The comment is a bit misleading, `timeout <= 0` is not the same as it not being provided. Just a reference to the specs should suffice.